src/tests/ScoreBoard.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, cleanup } from 'react-testing-library';
+import { ScoreBoard } from '../ScoreBoard';
+
+const originalLocalStorage = global.window;
+
+const mockLocalStorage = {
+  getItem: function (key) {
+    if (key === 'userMoves') {
+      return JSON.stringify([
+        { name: 'John', moves: 20 },
+        { name: 'Andy', moves: 8 },
+        { name: 'Bill', moves: 10 },
+      ])
+    }
+  },
+  setItem: jest.fn(),
+  clear: jest.fn()
+}
+
+beforeAll(() => {
+  global.localStorage = mockLocalStorage;
+})
+
+describe('ScoreBoard component', () => {
+  it('should render three items in list if three items are stored in localStorage', () => {
+    const { container } = render(<ScoreBoard />);
+    expect(container.querySelectorAll('li')).toHaveLength(3)
+  }) 
+
+  it('Should render three items in list if three items are stored in localStorage', () => {
```

Comment:
```suggestion
  it('should render three items in list if three items are stored in localStorage', () => {
```

As a convention everything in text description should be lowercase. Only technical terms like `localStorage` should be in camelCase. Only constants like `BOARD_SIZE_MAX` should be UPPERCASE.